\documentclass[11pt,letterpaper,pdf]{article}
\usepackage[T1]{fontenc}
\usepackage{fullpage}
\usepackage[utf8]{inputenc}
%% Hyperlinks and colors
\usepackage{color,hyperref, url}
\definecolor{darkblue}{rgb}{0.0,0.0,0.75}
\hypersetup{colorlinks,breaklinks,
  linkcolor=darkblue,urlcolor=darkblue,
  anchorcolor=darkblue,citecolor=darkblue}
%% Bibliography
\usepackage[autostyle]{csquotes}
\usepackage[backend=biber,
           bibstyle=numeric,sorting=ydnt,sortcites=true,natbib=true,defernumbers=true,
           maxbibnames=99,giveninits=true,uniquename=false]{biblatex}
\renewbibmacro{in:}{}
\addbibresource{hickey.bib}
\renewcommand*{\mkbibnamegiven}[1]{%
\ifitemannotation{highlight}
{\textbf{#1}}
{#1}}
% \ifitemannotation{highlight}
%   {\textbf{#1}}
%   {#1}
\renewcommand*{\mkbibnamefamily}[1]{%
\ifitemannotation{highlight}
  {\textbf{#1}}
  {#1}%
\ifitemannotation{first}
  {\textsuperscript{*}}
  {}%
\ifitemannotation{corresponding}
  {$^\dagger$}
  {}%
}%%
\usepackage{catchfile}
\newcommand{\getenv}[2][]{%
  \CatchFileEdef{\temp}{"|kpsewhich --var-value #2"}{}%
  \if\relax\detokenize{#1}\relax\temp\else\let#1\temp\fi}
\getenv[\INTERNALUSE]{INTERNALUSE}
\usepackage{changepage}
\newenvironment{myannotate}{\vspace{-\parskip}\begin{adjustwidth}{1cm}{}}{\end{adjustwidth}}
\usepackage{ifthen}
\usepackage{eqlist}
\usepackage{enumitem}
\setlength{\parindent}{0em}
\setlength{\parskip}{1ex plus0.5ex minus0.2ex}
\setlength{\fboxsep}{2.5pt}
\newcommand{\mycon}[1]{\smallskip\begin{enumerate}[resume,label={\scriptsize \arabic*$\ $},leftmargin=\parindent]\setlength{\itemsep}{#1}\vspace*{-0.7em}}
\newcommand{\ee}{\end{enumerate}}

\newcommand{\teach}[3]{%
  \vspace*{0.3\baselineskip}
  {#1}.\newline {\it #3}. #2.}

\newcommand{\talk}[4]{%
  \item #1. (#2) {\it #3} (#4).}

\newcommand{\poster}[4]{%
\item #1. (#2) {\it #3} (#4)}

% \newcommand{\grant}[7]{%
%   \vbox{%
%   \textbf{#5}\\
%   \vspace*{-1.8\baselineskip}
%   \begin{tabbing}
%     \= \hspace*{4cm} \= \hspace*{8cm} \= \kill
%     #1 \> \> #4 \> #3\\
%   \end{tabbing}
%   \vspace*{-1.8\baselineskip}
%   Principal Investigator: #2\\
%   Major Goals: #6\\
%   #7\\
%   \vspace*{5mm}
%   }
% }

\begin{document}

\hfill \today

\begin{center}
{\bf \Large CURRICULUM VITAE}\\
\vspace*{5mm}
{\Large Peter Francis Hickey}
\end{center}

% \vspace*{5mm}

\section*{PERSONAL DATA}

\begin{tabbing}
  \= Web Page:\hspace*{2cm}
  \=\href{https://www.peterhickey.org}{https://www.peterhickey.org}\\
  \> Email: \>\href{mailto:hickey@wehi.edu.au}{hickey@wehi.edu.au}\\
  \> Phone: \>+61-412-655-820\\
  \> Mailing Address: \>Advanced Technology and Biology\\
  \> \> Walter and Eliza Hall Institute of Medical Research\\
  \> \>1G, Royal Parade\\
  \> \>Parkville, VIC, 3052\\
\end{tabbing}

\section*{EDUCATION AND TRAINING}

\subsection*{Degrees}

\begin{tabbing}
  \=2015 \hspace*{1.5cm} \=Ph.D.\ in Statistics\\
  \>\>Department of Mathematics and Statistics\\
  \>\>The University of Melbourne, Melbourne \\
  \>\>Advisors: \textbf{Terry Speed} and \textbf{Peter Hall}\\
    \>2009\>B.\ Sc.\ (First Class Honours) in Mathematics and Statistics \\
    \>\>University of Melbourne
\end{tabbing}

\subsection*{Postdoctoral Training}

\begin{tabbing}
  \= 2016--2018 \hspace*{0.55cm} \= Department of Biostatistics \\
  \>\> Johns Hopkins Bloomberg School of Public Health\\
  \>\> Advisor: {\bf Kasper D.\ Hansen}
\end{tabbing}

\section*{PROFESSIONAL EXPERIENCE}

\begin{tabbing}
  \= 2018--Present \hspace*{0.1cm} \= Senior Research Officer\\
  \>\>Advanced Technology and Biology\\
  \>\>Walter and Eliza Hall Institute of Medical Research\\
  \> 2016--2018 \> Postdoctoral Fellow\\
  \>\>Department of Biostatistics\\
  \>\> Johns Hopkins Bloomberg School of Public Health\\
  \> 2010--2015 \> Research Assistant\\
  \>\>Bioinformatics Division\\
  \>\> Walter and Eliza Hall Institute of Medical Research
\end{tabbing}

\section*{PROFESSIONAL ACTIVITIES}

\subsection*{Professional Memberships}

\begin{tabbing}
  \= Member, Statistical Society of Australia\\
  \> Member, Australian Bioinformatics And Computational Biology Society\\
  \> Member, Australasian Genomic Technologies Association\\
\end{tabbing}

\section*{EDITORIAL ACTIVITIES}

\subsection*{Served as \textit{referee} for}

\begin{tabbing}
  \=Bioinformatics\\
  \>Biostatistics\\
  \>F1000Research\\
  \>Genetic Epidemiology\\
  \>Genome Biology\\
  \>Heredity\\
  \>Nature Methods\\
  \>PLoS Computational Biology\\
  \>PLoS Genetics\\
\end{tabbing}

\section*{HONORS AND AWARDS}

\begin{tabbing}
  \= 2019 \hspace*{0.5cm} \= Bioconductor Travel Award (To present at BioC in New York, USA)\\
  \= 2018 \> AGTA Travel Award (To present at the AGTA meeting in Adelaide, Australia)\\
  \= 2018 \> Bioconductor Travel Award (To present at BioC in Toronto, Canada)\\
  \= 2015 \> Bioconductor Travel Award (To present at BioC in Seattle, USA)\\
  \= 2015 \> Edith Moffat Travel Award (To interview for international for postdoctoral positions)\\
  \= 2013 \> Prize for best lightning talk at the Australian Epigenetics Conference\\
  \= 2013 \> Third prize for best lightning talk at the Young Statisticians Conference
\end{tabbing}

\section*{PUBLICATIONS}

\nocite{*}

\defbibnote{mynote}{$^*$ indicates equal contributions\\
  $^\dagger$ indicates corresponding author(s) (if not the senior author)\\
  % \textbf{boldface} indicates a member of my lab \\
  }

\printbibliography[title=Journal Articles (peer reviewed),keyword=peerjournal,prenote=mynote]

\printbibliography[title={Journal Articles, Consortia member (peer reviewed)},keyword=peerconsortia]

\printbibliography[title=Preprints (not peer reviewed),keyword=preprint,prenote=mynote]

\printbibliography[title={Theses, Editorials},keyword=others]

% NOTE: Currently not using prenote=mynote

% TODO: Add these.
% \printbibliography[title={Preprints, subsequently published (not peer reviewed)},keyword=pubpreprint,prenote=mynote]

\subsection*{Citation databases}

\vspace*{0.5\baselineskip}

Google Scholar: \href{https://scholar.google.com.au/citations?user=pQhJuagAAAAJ&hl=en}{profile} (link)\\
ORCID: \href{https://orcid.org/0000-0002-8153-6258}{0000-0002-8153-6258} (link)\\
Europe PMC Citations: \href{https://europepmc.org/authors/0000-0002-8153-6258}{profile} (link)

\section*{PRACTICE ACTIVITIES}

\subsection*{Software - Bioconductor Project}

\begin{enumerate}[labelindent=1cm,align=left]
  \item[\href{https://www.bioconductor.org/packages/bsseq}{bsseq}]
    Analyze, Manage and Store Bisulfite Sequencing Data.
  \item[\href{https://www.bioconductor.org/packages/CellBench}{CellBench}]
    Construct Benchmarks for Single Cell Analysis Methods.
  \item[\href{https://www.bioconductor.org/packages/DelayedMatrixStats}{DelayedMatrixStats}]
    Functions that Apply to Rows and Columns of 'DelayedMatrix' Objects.
  \item[\href{https://www.bioconductor.org/packages/GenomicTuples}{GenomicTuples}]
    Representation and Manipulation of Genomic Tuples.
  \item[\href{https://www.bioconductor.org/packages/MatrixGenerics}{MatrixGenerics}]
    S4 Generic Summary Statistic Functions that Operate on Matrix-Like Objects.
  \item[\href{https://www.bioconductor.org/packages/minfi}{minfi}]
    Analyze Illumina Infinium DNA Methylation Arrays.
  \item[\href{https://www.bioconductor.org/packages/tximeta}{tximeta}]
    Transcript Quantification Import with Automatic Metadata.
\end{enumerate}

\subsection*{Software - Other}

\begin{enumerate}[labelindent=1cm,align=left]
  \item[\href{https://pypi.python.org/pypi/methtuple/}{methtuple}] A caller for DNA methylation events that co-occur on the same DNA fragment from high-throughput bisulfite sequencing data, such as whole-genome bisulfite-sequencing.
\end{enumerate}

\ifthenelse{\equal{\INTERNALUSE}{TRUE }}{

\newpage

\begin{center}
{\bf CURRICULUM VITAE}\\
Peter Francis Hickey\\[3mm]
Part II
\end{center}

\bigskip

\subsection*{LEADERSHIP AND MANAGEMENT}

\subsection*{Staff Supervision}

\begin{tabbing}
 \=William Ho, Senior Research Officer, SCORE, WEHI, 2019--present.\\
\end{tabbing}

\subsection*{Conference Organisation}

\begin{tabbing}
 \=Organiser, BioC Asia 2020 (Virtual Conference).\\
 \>Organiser, BioC Asia 2019 (Sydney, Australia).\\
 \>Organising Committee, BioC 2019 (New York, USA).\\
 \>Organiser, Oz Single Cell 2019 (Melbourne, Australia).\\
\end{tabbing}

\subsection*{TEACHING}

\subsection*{Ph.D.\ Supervision}

\begin{tabbing}
 \=Yue You (co-supervised with Matt Ritchie), Medical Biology, WEHI, 2020--present.\\
 \>Shian Su (co-supervised with Matt Ritchie), Medical Biology, WEHI, 2020--present.\\
\end{tabbing}

\subsection*{Undergraduate Supervision}

\begin{tabbing}
  \=Amelia Dunstone, Undergraduate Research Opportunities Program, 2019--present.\\
\end{tabbing}

\subsection*{Ph.D.\ Committee}

\begin{tabbing}
  \=Aravind Manda, Population Health and Immunity, 2020--present.\\
  \>Megan Iminitoff, Epigenetics and Development Division, 2019--present.\\
\end{tabbing}

\subsection*{Classroom Instruction - Invited Guest Lecturer}

\begin{tabbing}
  \=Introduction to Single-Cell 'Omics: University of Melbourne, 2019.\\
  \>Analysis of ATAC-seq data: Johns Hopkins Bloomberg School of Public Health, 2017.
\end{tabbing}

\subsection*{Other significant teaching - Workshops and Short Courses}

\teach{Effectively using the DelayedArray framework to support the analysis of large datasets}{2020}{BioC, Virtual Conference}

\teach{Experimental Design and Statistics}{2020}{WEHI, Melbourne, Australia}

\teach{Single-cell RNA-seq workshop (2-day workshop)}{2020}{WEHI, Melbourne, Australia}

\teach{Building a Bioconductor package (co-taught with Saskia Freytag)}{2019}{BioC Asia, Sydney, Australia}

\teach{Hands on workshop on downstream analysis of 10X data}{2019}{Oz Single Cell 2019, Melbourne, Australia}

\teach{Effectively using the DelayedArray framework to support the analysis of large datasets}{2019}{BioC, New York, USA}

\teach{Experimental Design and Statistics}{2019}{WEHI, Melbourne, Australia}

\teach{Analyzing 10X Chromium single-cell RNA-seq}{2018}{University of Melbourne, Melbourne, Australia}

\teach{Analyzing 10X Chromium single-cell RNA-seq}{2018}{Nanjing University, Nanjing, China}

\teach{Effectively using the DelayedArray framework to support the analysis of large datasets}{2018}{BioC, Toronto, Canada}

\teach{Analysing DNA methylation data with Bioconductor}{2016}{BioC, Palo Alto, USA}

% \section*{RESEARCH GRANT PARTICIPATION}

% \subsection*{Ongoing}

% \subsection*{Completed}

% \section*{ACADEMIC SERVICE}

\section*{PRESENTATIONS}

% \subsection*{Upcoming}

% \mycon{0.3em}
% \talk{empty}{}{}{}
% \ee

\subsection*{Invited Talks (Seminars and Scientific Meetings)}

% TODO: Link to slides?

\mycon{0.3em}

\talk{Overview of single-cell bioinformatics: from data to biology}{2020}{CSL}{Melbourne, Australia}

\talk{Overview of single-cell bioinformatics}{2019}{Oz Single Cell}{Melbourne, Australia}

\talk{How UROP kickstarted my scientfic career}{2019}{UROP Welcome Forum}{Melbourne, Australia}

\talk{Data Organisation: Making Your Research Life Easier}{2019}{Australian Catholic University}{Melbourne, Australia}

\talk{Bioinformatics for bisulfite sequencing}{2013}{La Trobe University}{Melbourne, Australia}

\talk{X chromosome association testing in genome wide association studies}{2010}{Statistical Society of Australia - Victorian branch meeting}{Melbourne, Australia}

\ee

\subsection*{Scientific Meetings (Contributions)}

% TODO: Link to slides?

\mycon{0.3em}

\talk{Using the DelayedArray framework for processing large datasets}{2018}{BioC Asia}{Melbourne, Australia}

\talk{Getting help and helping others (including future you)}{2018}{BioC Asia}{Melbourne, Australia}

\talk{Genome-wide analysis of DNA methylation in samples from the Genotype-Tissue Expression (GTEx) project}{2018}{AGTA}{Adelaide, Australia}

\talk{DelayedArray: A tibble for arrays}{2018}{useR!}{Brisbane, Australia}

\talk{Lessons from switching to on-disk storage using DelayedArray containers}{2018}{BioC}{Toronto, Canada}

\talk{Mapping the human brain epigenome and its links to disease}{2017}{Epigenetics Australia}{Brisbane, Australia}

\talk{Neuronal brain region-specific DNA methylation and chromatin accessibility are associated with neuropsychiatric disease heritability}{2017}{GTEx Project Community Meeting}{Rockville, USA}

\talk{Scaling R and Bioconductor to support methods for single-cell genomic analysis}{2017}{JSM}{Baltimore, USA}

\talk{Developing statistical methods for large epigenomic studies in the human brain}{2017}{ENAR Spring Meeting}{Washington D.C., USA}

\talk{DelayedMatrixStats: Porting the matrixStats API to work with DelayedMatrix objects}{2017}{BioC}{Boston, USA}

\talk{New features in `bsseq' for analysing large whole genome bisulfite-sequencing datasets}{2016}{BioC}{Palo Alto, USA}

\talk{The `GenomicTuples' package}{2015}{BioC}{Seattle, USA}

\talk{Genomic tuples and DNA methylation patterns}{2015}{BioC Europe}{Heidelberg, Germany}

\talk{Simulating whole-genome DNA methylation data}{2014}{Australian Statistical Conference / International Mathematical Statistics Annual Meeting}{Sydney, Australia}

\talk{Exploiting local dependencies in genome-wide studies of DNA methylation}{2013}{Young Statisticians Conference}{Melbourne, Australia}

\talk{Spatial dependence of CpG-methylation from whole genome bisulfite sequencing}{2012}{Epigenomics of Common Diseases Meeting}{Baltimore, USA}

\talk{Spatial dependence of DNA methylation}{2012}{Australian Statistical Conference}{Adelaide, Australia}

\talk{Bioinformatics - Applied statistics in modern molecular biology}{2010}{Victorian Mathematics and Statistics Studentsâ€™ Conference}{Melbourne, Australia}

\ee

\subsection*{Other}

\mycon{0.3em}

\talk{Simplified Science: Single-cell RNA-sequencing}{2019}{WEHI Simplified Science}{Melbourne, Australia}

\talk{Making sense of DNA methylation data}{2014}{WEHI PhD Completion Seminar}{Melbourne, Australia}

\ee

\subsection*{Posters}

% TODO: Link to posters?

\mycon{0.3em}

\poster{Genome-wide analysis of DNA methylation in samples from the Genotype-Tissue Expression (GTEx) project}{2019}{Lorne Genome}{Lorne, Australia}

\poster{Developing `standard' bioinformatics analyses for the Single Cell Open Research Endeavour (SCORE)}{2018}{ABACBS}{Melbourne, Australia}

\poster{Simulating whole-genome bisulfite-sequencing data}{2014}{Lorne Genome}{Lorne, Australia}

\poster{Simulating whole-genome bisulfite-sequencing data}{2013}{Epigenetics Australia}{Shoal Bay, Australia}

\poster{Analysis of mouse exome sequencing: filtering institute-specific single nucleotide variants (SNVs)}{2011}{GeneMappers}{Hobart, Australia}

\poster{X chromosome association testing in genome wide association studies}{2010}{The International Genetic Epidemiology Society Conference}{Boston, USA}

\poster{X chromosome association testing in genome wide association studies}{2010}{The Australasian Microarray and Associated Technologies Association Conference}{Hobart, USA}

\poster{Homozygosity by state analysis in highly inbred pedigrees}{2009}{GeneMappers}{Sydney, USA}

\ee

}{}
% END IF THEN

\end{document}

% Local Variables:
% LocalWords: LocalWords Biostatistics WEHI JHU
% End:
